tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTests.java

```diff
@@ -177,6 +178,7 @@ public void compensateMultiLevelNestedActivity() throws WebApplicationException
     }
 
     @Test
+    @Ignore
```

Comment:
Why isn't this included in the PR for issue #274  ?

Reply:
Assuming it is required for this to be ignored to get this PR to pass, might I suggest a comment in the code and on the commit message as to why it is ignored? and associate this PR with both issues (even if it is just to ignore this test and doesn't fix 274)

Reply:
Yes, this is the conflict described in #274. I will do what Tom suggests.